sztp-agent/pkg/secureAgent/daemon.go

```diff
@@ -7,12 +7,44 @@ Copyright (C) 2022 Red Hat.
 
 package secureAgent
 
-func RunCommandDaemon() error {
-	a := NewAgent("")
-	err := a.execDaemon()
+import (
+	"errors"
+	"log"
+	"os"
+)
+
+const (
+	DHCLIENT_LEASE_FILE = "/var/lib/dhclient/dhclient.leases"
```

Comment:
we might do it a separate file so we can have logic for both: DHCP, mDNS and SLAAC
wdyt ?

Reply:
WDYM? 
instead of the loop looking this:
```
if _, err := os.Stat(DHCLIENT_LEASE_FILE); err == nil {
		line := linesInFileContains(DHCLIENT_LEASE_FILE, SZTP_REDIRECT_URL)
		a.BootstrapURL = extractURLfromLine(line, `(?m)[^"]*`)
		log.Println(a)
```
have a function to look 3 sources (dhclient.leases, slaac and mdns) to know where's the change and continue with this change?